crates/core/src/constraint_system/validate.rs

```diff
@@ -31,8 +32,58 @@ where
 	P: PackedField<Scalar = F> + PackedExtension<BinaryField1b>,
 	F: TowerField,
 {
+	let ConstraintSystem {
+		oracles,
+		table_constraints,
+		non_zero_oracle_ids,
+		flushes,
+		channel_count,
+		table_size_specs,
+		exponents: _,
```

Comment:
It seems that we never really implemented validation for `exponents`. Subject for another issue.

Reply:
yes, but it is in TODO

Reply:
Yeah, I know, I added that issue : )